phantombot/umbrel-app.yml

```diff
@@ -0,0 +1,36 @@
+manifestVersion: 1.1
+id: phantombot
+category: social
+name: PhantomBot
+version: "3.14.1.0"
+tagline: PhantomBot is a Twitch chat bot powered by Java
+icon: ""
+gallery: []
+description: >-
+  PhantomBot is an actively developed open source interactive Twitch bot with a vibrant community that provides entertainment and moderation for your channel,
+  allowing you to focus on what matters the most to you - your game and your viewers.
+
+
+  🛠️ Set-Up Instructions
+   Required! If you don't perform the initial setup, you will encounter an error when connecting to the control panel!
+
+  1. Connect Your Twitch Account
+     Using the instructions on the page https://umbrel.local:25000/oauth/, create oauth app and connect your Twitch account.
+
+  2. Fill Chanel Configuration Fields
+     Complete the channel and owner fields on the configuration page https://umbrel.local:25000/setup/.
+
+  3. Log In
+     Done! You can log in to the control panel at https://umbrel.local:25000/panel/login using the default credentials.
```

Comment:
Awesome work adding these!

If users can navigate to these links by clicking things within the UI, then I would suggest making these instructions more generic. For example, telling the user to "navigate to the oauth settings" instead of providing a specific link.

My reasoning here is that:
1) These links may change in future app updates and it will be difficult to remember to update the app description

2) It's possible that the user accesses their umbrel at a different hostname than `umbrel.local`. For example, they may access their umbrel via local IP address, or by Tailscale, or if they have multiple Umbrel devices they may be using umbrel-2.local